core/express-server/package.json

```diff
@@ -35,6 +35,7 @@
   "devDependencies": {
     "@itwin/build-tools": "workspace:*",
     "@itwin/core-common": "workspace:*",
+    "@itwin/core-backend": "workspace:*",
```

Comment:
This would need to be a peerDependency the way it's written today. Maybe we could get away with making it an optional peerDependency though since it's only required in the WebEdit case.